app/controllers/concerns/saml_idp_auth_concern.rb

```diff
@@ -94,6 +95,7 @@ def saml_response
       reference_id: active_identity.session_uuid,
       encryption: current_service_provider.encryption_opts,
       signature: saml_response_signature_options,
+      signed_response_message: current_service_provider.signed_response_message_requested,
```

Comment:
in order for this to work, we need a version bump inside `Gemfile.lock` right?

Reply:
Yes, once the `saml_idp` gem is merged.